featurebyte/service/offline_store_feature_table_manager.py

```diff
@@ -215,6 +223,12 @@ async def handle_online_enabled_features(self, features: List[FeatureModel]) ->
                     feature_table_model
                 )
 
+            if update_progress:
+                await update_progress(
+                    int((idx + 1) / offline_table_count * 90),
+                    f"Materializing features to online store for table {offline_store_table_name}",
+                )
+
         feature_store_model = await self._get_feature_store_model()
         new_tables.extend(
             await self._create_or_update_entity_lookup_feature_tables(
```

Comment:
I think good be good to have a progress message indicating that we are starting to update these tables since there can be quite a few of them.

Reply:
Agree, this can takes more time then other offline table when running it locally for 292 features. Is it ok if i do the following allocation:

0 - 60 => offline feature table update
60 - 90 => entity lookup table update
90 => 100 comment update

Reply:
The ideal split depends on the number of entity lookup tables, but I think this is already a good improvement.

Reply:
Yes. Currently 60-90 will be uniformly split based on number of entity lookup tables.